selfdrive/car/gm/interface.py

```diff
@@ -212,7 +212,9 @@ def _update(self, c):
     if ret.cruiseState.standstill:
       events.add(EventName.resumeRequired)
     if ret.vEgo < self.CP.minSteerSpeed:
-      events.add(car.CarEvent.EventName.belowSteerSpeed)
+      events.add(EventName.belowSteerSpeed)
+    if ret.vEgoRaw <= 0.5 and ret.brake <= 20:
+      events.add(EventName.depressBrakePedal)
```

Comment:
The true condition is `ECMVehicleSpeed->VehicleSpeed` != 0, but it becomes non-zero around when vEgoRaw is 0.5 m/s.

The brake pressed threshold is also either 19 or 20. Faults at < 0.5 m/s slowly increasing pressure until you pass this threshold